docs/usage/maintain-shoot.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Shoot Cluster Maintenance
```

Comment:
If there are some differences compared to https://github.com/gardener/gardener/blob/master/docs/usage/shoot_maintenance.md, please add them in the existing file, but for sure we do not want to have duplicated documents and information.

Reply:
Thanks for pointing that out!
Unfortunately we have already here and there duplicated documents. Yet another reason to keep the documentation in the corresponding component (and in one repo). Until now we allowed the content in the central documentation repo to run its own course.